performance/client/src/__tests__/primary/ExplorerAPIs/get-job-details.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+
+import WrkTestCase from "../../../testcase/wrk";
+import { getBasicAuthorizationHeader } from "../../../utils";
+
+class ExplorerApiJobDetailsTest extends WrkTestCase {
+  fetchZoweVersions = true;
+
+  name = "Test explorer data sets api endpoint /api/v2/jobs/{jobName}/{jobId}";
+  endpoint = '/api/v2/jobs/SDSF/STC01060';
```

Comment:
I think SDSF should be ok, but the job ID STC01060 may not exist on other systems. We will need to make a API call to list jobs IDs of SDSF first, and pick the active job.

Reply:
I've modified this test case to first make an API call to retrieve the job ID for the active SDSF job on the system and then use that ID in running the test case.